### PR TITLE
Finish biopython/bioconductor sections, add initial 'installing julia/biojulia' sections

### PR DESCRIPTION
**Pull request by M-PERSIC**

### CODE REVIEW

File: docs/make.jl
Comment (kescobo) [inline docs/make.jl:1]: Can you add `push_preview = true` to `deploydocs()` call? https://github.com/BioJulia/BioJuliaDocs/actions/runs/5746103087/job/15575087469#step:5:50

See also https://documenter.juliadocs.org/dev/man/hosting/#gh-pages-Branch

**Review by kescobo — COMMENTED** [review]
See below - would be nice to be able to read on the web to review

File: docs/src/comparisons/bioconductor.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# [Bioconductor (R)](@id bioconductor)
+
+- [Bioconductor](https://www.bioconductor.org/) represents an ecosystem of related genomics tools written in the R programming language
+- Tools written in other languages, such as C and Python, are also included with R wrappers available 
+- Released over two decades ago and hosts over 2000 packages for bioinformatics and related fields
```
Comment (kescobo) [inline docs/src/comparisons/bioconductor.md]:
```suggestion
- Initially released over two decades ago, it now hosts over 2000 packages for bioinformatics and related fields
```

File: docs/src/comparisons/bioconductor.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| High-level, general-purpose compiled language                          | High-level, interpreted language for statistical computing                       |
+| Dynamically typed with multiple dispatch and optional type annotations | Dynamically typed without type annotations support                               |
+| Built-in parallelism via threads, coroutines (Tasks)                   | Parallelism via external libraries (BLAS, parallel,...)                          |
+| Extensive metaprogramming (Lisp-like macros, generated functions,...)  | Less extensive metaprogramming (function factories, expression manipulation,...) |
+| Single implementation available (JuliaLang)                            | Multiple implementations available (pqR, Renjin,...)                             |
+
+
+**To transition from R to Julia:**
+- See [Noteworthy differences from R](https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-R) in the Julia manual for a more in-depth comparison
+- Use [RCall.jl](https://github.com/JuliaInterop/RCall.jl.git) to seamlessly integrate R code into your Julia project
```
Comment (kescobo) [inline docs/src/comparisons/bioconductor.md]: Maybe add https://cran.r-project.org/web/packages/JuliaCall/readme/README.html

File: docs/src/comparisons/biopython.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+| Built-in parallelism via threads, coroutines (Tasks)                   | Limited (GIL) parallelism via threads, async/await                       |
+| Extensive metaprogramming (Lisp-like macros, homoiconicity,...)        | Less extensive metaprogramming (decorators, metaclasses,...)             |
+| Included default package manager                                       | No default package manager (poetry, conda, other alternatives available) |
+| Single implementation available (JuliaLang)                            | Multiple implementations available (PyPy, Numba,...)                     |
+| Arrays are column-major (columns are contiguous in memory)             | (Numpy) arrays are row-major by default (rows are contiguous in memory)  |
+
+**To transition from Python3 to Julia:**
+- See [Noteworthy differences from Python](https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-Python) in the Julia manual for a more in-depth comparison
+- Use [PythonCall.jl](https://github.com/cjdoris/PythonCall.jl.git)/[CondaPkg.jl](CondaPkg.jl) to seamlessly integrate Python3 code into your Julia project
```
Comment (kescobo) [inline docs/src/comparisons/biopython.md:43]: add https://pypi.org/project/juliacall/

File: docs/src/getting_started/juliainstallation.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+```
+# First option: Install juliaup via curl
+FROM ubuntu:latest
+
+RUN apt update && \
+    apt install -y curl && \
+    curl -fsSL https://install.julialang.org | sh -s -- -y # install juliaup non-interactively
+
+RUN juliaup add 1.6
```
Comment (kescobo) [inline docs/src/getting_started/juliainstallation.md]:
```suggestion
RUN juliaup add release
```

**Review by kescobo — APPROVED** [review]
Looking good!

File: docs/make.jl
Comment (M-PERSIC) [reply docs/make.jl:1]: Just added, I'll try another small pr later tonight both to add new content and test it out to see if it works alright.

**Comment by M-PERSIC** [comment]
All set, merge when ready!

**Pull request merged**